README.md

```diff
@@ -193,14 +209,31 @@ try {
 }
 ```
 
-Where `yourUserSearchMethod` is a piece of logic in your app that is supposed to find a user, given a userId.
-No matter if the user is a new or an existing one, Yoti will always provide her/his profile, so you don't necessarily need to store it.
+Where `yourUserSearchMethod` is a method in your app that finds the user for the given userId.
+Regardless of whether the user is new to your app, Yoti will always provide the profile.  So you don't necessarily need to store it.
+
+### Attributes
+
+User details are provided by a list of attributes on the `com.yoti.api.client.HumanProfile` class.  Whether a given attribute is present or not depends on the settings you have applied on Yoti Dashboard.
+
+Attributes are returned as an instance of `com.yoti.api.client.Attribute<T>`.  Since v2.0.0 of the SDK, Attribute is a generic class where <T> represents the type of the Attribute value returned.
+
+### Anchors, Sources and Verifiers
 
-The `com.yoti.api.client.HumanProfile` class provides a set of methods to retrieve different user attributes. Whether the attributes are present or not depends on the settings you have applied to your app on Yoti Dashboard.
+`com.yoti.api.client.Anchor` represents how a given `Attribute<T>` has been _sourced_ or _verified_.  These values are created and signed whenever a Profile Attribute is created, or verified with an external party.
+
+For example, an attribute value that was _sourced_ from a Passport might have the following values:
+| attribute.getType | SOURCE |
+| attribute.getValue | PASSPORT |
+| attribute.getTimestamp | 2017-10-31, 19:45:59.123789 |
```

Comment:
Since type, value & timestamp are associated with an anchor and not an attribute, shouldn't this be something like:

For example, an attribute value that was _sourced_ from a Passport might have the following **anchor properties**:

and then:
attribute.**anchor**.getType,
attribute.**anchor**.getValue,
attribute.**anchor**.getTimestamp?

Reply:
Correct.  That's wrong.  Fixed.